plugin-maven/CHANGES.md

```diff
@@ -4,7 +4,14 @@ We adhere to the [keepachangelog](https://keepachangelog.com/en/1.0.0/) format (
 
 ## [Unreleased]
 ### Fixed
-* Remove Java files from default Maven Groovy formatting ([#1051](https://github.com/diffplug/spotless/pull/1051)).
+* **BREAKING** Remove Java files from default Maven Groovy formatting ([#1051](https://github.com/diffplug/spotless/pull/1051)).
+  * This breaking change affects you only if you are using the `<groovy>` language block **and** you have `.java` files located within `src/main/groovy` or `src/test/groovy`.
+  * Before `3.0.0`, the default target of groovy was
+    * `src/main/groovy/**/*.groovy`
+    * `src/test/groovy/**/*.groovy`
+    * `src/main/groovy/**/*.java`
```

Comment:
Just a small correction. I removed the following file masks:
 * `src/main/java/**/*.java`
 * `src/test/java/**/*.java`

The Groovy formatter tried to format Java sources that live under the Java sources root. I think it is weird and perhaps even undesirable to format such source files with a Groovy formatter. This PR changes the default behavior, but it is more like a fix in my opinion.

Reply:
Oh man that is super weird! It's so weird my brain couldn't read it the first time, thanks for pointing that out! I always agreed that it was a fix, but now I also agree that it is not a "breaking fix".